atlasreader/atlasreader.py

```diff
@@ -673,21 +670,34 @@ def create_output(filename, atlas='all', voxel_thresh=1.96, cluster_extent=20,
                                   cmap=plotting.cm.cold_hot,
                                   output_file=glass_fname)
 
-    # get template image for plotting cluster maps
-    bgimg = nb.load(
-        resource_filename(
-            'atlasreader',
-            'data/templates/MNI152_T1_1mm_brain.nii.gz'
+    # Check if thresholded image contains only zeros
+    if thresh_img.get_data().sum():
```

Comment:
Basically impossible, but what if the data contain positive and negatives that _exactly_ sum to 0? It might be nice to instead just check for the existence of non-zeros!

Reply:
You're completely right. I'm getting tired... I mean I use `np.count_nonzero` already just before.